### PR TITLE
[ID] Update `Mapping and modding timeline`

### PR DESCRIPTION
**Pull request by Nivalyx**
An update for the Indonesian translation of the `Mapping and modding timeline` (http://osu.ppy.sh/wiki/en/Mapping_and_Modding_Timeline) wiki article.

---

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Mapping_and_Modding_Timeline/id.md [hunk 1]
```diff
@@ -1,11 +1,6 @@
----
-outdated: true
-outdated_since: 4f8d54a3fc70074630230b88709f573a969c9320
----
-
 # Sejarah mapping dan modding
 
-Sistem mapping dan modding yang berlaku di dalam lingkungan osu! senantiasa berkembang seiring waktu. Artikel **Sejarah mapping dan modding** di bawah ini ditujukan sebagai suatu bentuk dokumentasi terhadap berbagai perubahan yang terjadi dari masa ke masa dan sebagai acuan bagi mereka yang ingin mencetuskan ide-ide baru terkait mapping dan modding ke depannya.
+Sistem mapping dan modding yang berlaku di dalam lingkungan osu! senantiasa berubah seiring waktu. Artikel **Sejarah mapping dan modding** di bawah ini ditujukan sebagai suatu bentuk napak tilas akan berbagai perubahan yang terjadi dari masa ke masa dan sebagai pedoman bagi mereka yang ingin mencetuskan ide-ide baru untuk mengembangkan sistem mapping dan modding yang telah ada ke depannya.
```
Comment (ZeroPyrozen) [inline wiki/Mapping_and_Modding_Timeline/id.md:3]:
```suggestion
Sistem mapping dan modding yang berlaku di dalam lingkungan osu! senantiasa berubah seiring waktu. Artikel **Sejarah mapping dan modding** di bawah ini ditujukan sebagai suatu bentuk tapak tilas akan berbagai perubahan yang terjadi dari masa ke masa dan sebagai pedoman bagi mereka yang ingin mencetuskan ide-ide baru untuk mengembangkan sistem mapping dan modding yang telah ada ke depannya.
```
Formal word.

File: wiki/Mapping_and_Modding_Timeline/id.md [hunk 2]
```diff
@@ -393,15 +388,15 @@ Secara keseluruhan, terdapat **1847 beatmap** yang di-rank sepanjang tahun 2017.
 
 ### Desember
 
-- **2018-12-12:** tim inti osu! secara resmi [menjauhkan diri dari praktek komisi beatmap (*beatmap commission*)](https://osu.ppy.sh/community/forums/topics/840838) yang tidak beritikad baik.
+- **2018-12-12:** Tim inti osu! menyatakan bahwa [osu! secara resmi menjauhkan diri dari praktek komisi beatmap (*beatmap commission*)](https://osu.ppy.sh/community/forums/topics/840838) yang tidak beritikad baik.
```
Comment (ZeroPyrozen) [inline wiki/Mapping_and_Modding_Timeline/id.md:391]:
```suggestion
- **2018-12-12:** Tim inti osu! menyatakan bahwa [osu! secara resmi menjauhkan diri dari praktik komisi beatmap (*beatmap commission*)](https://osu.ppy.sh/community/forums/topics/840838) yang tidak beritikad baik.
```
Formalise `praktek` into `praktik`

**Review by ZeroPyrozen — CHANGES_REQUESTED** [review]

**Comment by Nivalyx** [comment]
@ZeroPyrozen applied both, thanks for the check 👍 

(also changed all other instances of `praktek` throughout the article to `praktik`)

**Review by ZeroPyrozen — APPROVED** [review]
All good 👍

**Review by TicClick — APPROVED** [review]

**Pull request merged**